config/config.debug.toml

```diff
@@ -27,6 +27,17 @@ Height = 32
 [BridgeServer]
 GRPCPort = "9090"
 HTTPPort = "8080"
+DefaultPageLimit = 25
+MaxPageLimit = 100
+BridgeVersion = "v1"
+    [BridgeServer.DB]
+    Database = "postgres"
+    User = "test_user"
+    Password = "test_password"
+    Name = "test_db"
+    Host = "localhost"
+    Port = "5435"
```

Comment:
Why 5435?? Using this value, I think we should modify the docker compose file

Reply:
for the `localhost`, we should use the binding port, right?